static/sass/_pattern_strip.scss

```diff
@@ -1023,3 +1024,23 @@
     color: $color-x-light;
   }
 }
+
+@mixin ubuntu-p-strip-whitesuru {
+  .p-strip--whitesuru {
+    @extend %vf-strip;
+
+    background: url("https://assets.ubuntu.com/v1/25538a5e-Suru%20top%20left%20angle.jpg");
+    background-position: top left;
+    background-repeat: no-repeat;
+    background-size: contain;
+
+    // Hide suru on small and medium screens
+    @media screen and (max-width: $breakpoint-small - 1) {
```

Comment:
do we always subtract 1 from the breakpoints? Or is it better to overwrite the breakpoint variables to be smaller by 1 px?

Reply:
This convention comes from the [Vanilla documentation](https://vanillaframework.io/docs/settings/breakpoint-settings) so, ideally, we should always follow that as it's way easier for future maintenance. Updating the breakpoint variable on u.com only would mean that if there's ever a future Vanilla update that touches that variable, the u.com version will be out of sync with base Vanilla and will also need to be updated which can get pretty messy